python/mxnet/ndarray/random.py

```diff
@@ -431,3 +431,35 @@ def multinomial(data, shape=_Null, get_prob=False, out=None, **kwargs):
     <NDArray 2 @cpu(0)>
     """
     return _internal._sample_multinomial(data, shape, get_prob, out=out, **kwargs)
+
+
+def shuffle(data, **kwargs):
```

Comment:
Python interface seems unnecessary. You can register the operator with name _random_shuffle in C++.

Reply:
Still valid @reminisce ?